app/models/pawn.rb

```diff
@@ -17,38 +17,45 @@ def can_promote?(y_pos)
   end
   
   #logic to perform enpassant
-  def enpassant
-    if is_white?
-      self.update_attributes(x_pos: @opponent_pawn.x_pos, y_pos: @opponent_pawn.y_pos + 1)
+  def move_to!(to_x, to_y)
+    if can_enpassant?
+    to_x = @opponent_pawn.x_pos
+      if is_white?
+        diagonal_move_one_square?(to_x, 2)
+        capture_piece!(to_x, 2, @opponent_pawn)
+      else is_black?
```

Comment:
No statements are necessary on `else` you could remove `is_black?` or change it to `elsif` if that parameter is required. That should fix the code climate issue.